### PR TITLE
Feat : Added Basic Routing

### PR DESCRIPTION
**Pull request by srejitk**
This Feature Merge adds the following

1. Basic Folder Structure
2. Mockbee Setup
3. Mockman Setup
4. Basic Routing

### CODE REVIEW

**Review by shuntzhere — APPROVED** [review]
Looks clean, no change required.

File: House-of-glass/src/pages/api-response/index.jsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import MockApi from "./ApiResponse";
+
+export { MockApi };
```
Comment (vaibhav-mahalle) [inline House-of-glass/src/pages/api-response/index.jsx:3]: liked the precise commit messages as well as precise commits.

File: House-of-glass/public/index.html [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <!--
       Notice the use of %PUBLIC_URL% in the tags above.
```
Comment (vaibhav-mahalle) [inline House-of-glass/public/index.html]: You could remove the commented code if no longer required.

File: House-of-glass/src/router/Router.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Routes,Route } from 'react-router-dom';
+import { MockApi } from '../pages/api-response';
+import Cart from '../pages/cart/Cart';
+import Landing from '../pages/landing/Landing';
+import Products from '../pages/products/Listing/Products';
+import NotFound from '../pages/utils/NotFound/NotFound';
+import Wishlist from '../pages/wishlist/Wishlist';
```
Comment (vaibhav-mahalle) [inline House-of-glass/src/router/Router.jsx]: You could make it cleaner by importing all the components in index.js and from there importing all the components using single import statement.

**Review by vaibhav-mahalle — APPROVED** [review]
clean and concise code suggested some improvements please look into them.

File: House-of-glass/src/pages/api-response/index.jsx (hunk 1 above)
Comment (srejitk) [reply House-of-glass/src/pages/api-response/index.jsx:3]: Thank You

File: House-of-glass/src/router/Router.jsx (hunk 3 above)
Comment (srejitk) [reply House-of-glass/src/router/Router.jsx]: Thank You Vaibhav, Have corrected them in the latest commit.

File: House-of-glass/public/index.html (hunk 2 above)
Comment (srejitk) [reply House-of-glass/public/index.html]: Thank You Vaibhav, Have corrected them in the latest commit.

**Pull request merged**